content/05.main-text.md

```diff
@@ -1,9 +1,60 @@
 ## Introduction {.page_break_before}
 
+A *network* (also known as a [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics))) is a conceptual representation of a group of things — called *nodes* — and the relationships between them — called *edges*.
+Typically, a network has only one type of node and one type of edge.
+But in many cases, it is necessary to be able to distinguish between different types of entities and relationships.
+
+### Hetnets
+
+A *hetnet* (short for **het**erogeneous information **net**work [@doi:10.15363/thinklab.d104]) is a network where nodes and edges can be multiple types.
```

Comment:
Would you say it's one where they can be multiple types or would you say it only describes those where there are multiple types of either or both nodes/edges?

Reply:
I would say a hetnet requires nodes and edges to have type. A general implementation would support a graph with a single node and edge type. There's not any reason to use a hetnet in this case. 

Tweaked in https://github.com/greenelab/connectivity-search-manuscript/pull/5/commits/cc608e695b788595883a89f12f389e905977020d, so this is less of an issue.